docs/versioned_docs/version-zenith/quickstart/883939-containers.mdx

```diff
@@ -36,17 +36,17 @@ You can chain these functions together to build and return a Wolfi container ima
 
 This means that the build succeeded, and a `Container` type representing the built container image was returned. And, like `Directory`, `Container` is a Dagger core type which comes with useful functions of its own.
 
-### Start an interactive shell session with the container
+### Start an interactive terminal session with the container
 
-One of the most interesting `Container` functions is `Shell()`, which can be used to open an interactive shell session with the container. This feature is very useful for debugging and experimenting since it allows you to interact with containers directly and at any stage of your Dagger function execution.
+One of the most interesting `Container` functions is `Shell()`, which can be used to open an interactive terminal session with the container. This feature is very useful for debugging and experimenting since it allows you to interact with containers directly and at any stage of your Dagger function execution.
```

Comment:
```suggestion
One of the most interesting `Container` functions is `Terminal()`, which can be used to open an interactive terminal session with the container. This feature is very useful for debugging and experimenting since it allows you to interact with containers directly and at any stage of your Dagger function execution.
```